src/components/Tag/examples/default.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import Tag from '../Tag';
+
+export default React.createClass({
+	render() {
+		return (
+			<section>
+				<section>
```

Comment:
There is nothing wrong with having a `<section>` inside a `<section>` but I expect a section to carry unique content. In this case I would replace the inner section tags with `<article>` instead.